scripts/userstyles.yml

```
@@ -562,6 +565,12 @@ userstyles:
     readme:
       app-link: "https://www.perplexity.ai"
       current-maintainers: [*isabelroses, *tnixc]
+  picrew:
+    name: Picrew
```

To fix: https://github.com/catppuccin/userstyles/pull/671#issuecomment-1995919382, you need to add a color here, like so:
```suggestion
    name: Picrew
    color: text
```